general/design-dev-cycle.md

```diff
@@ -0,0 +1,19 @@
+Product/Design development cycle for source{d} application:
+
+- a) Team Sport
+- b) Problem Discovery
+- c) Ideation & Validation (low-fi interactive draft presentation & discussion)
+- d) Wireframing (scoped features interactive design asset)
+- e) Wireframes User Testing (scripted & recorded proto-usabllity tests)
+- f) Research Analysis & Conclusions
+- g) Wireframes Iteration (asset ready for implementation)
+- h) Development Roadmap/Planning & Metrics Assessment Definition
+- i) Raw UI Implementation
+- j) UI Visual Design in-browser (css or sass will be delivered for implementation)
+- k) Cycle trough i & j until planned objectives are met
+- l) Version Release User Testing (scripted & recorded proto-usabllity tests)
+- m) Research Analysis & Conclusions
+- n) Version Release Improvements
+- o) Ship Product
+- p) Evaluate Metrics
+- q) Start over from a
```

Comment:
Once we're settled with the items/order I can make it into a editable https://www.draw.io/ flowchart to make for easier reading.